tasks.py

```diff
@@ -98,7 +95,7 @@ def qa(ctx):
         else:
             success('No lint to fix')
         info('Ensure PyPI can render README and CHANGELOG')
-        readme_results = ctx.run('python setup.py check -m -s', pty=True, warn=True, hide=True)
+        readme_results = ctx.run('python3 setup.py check -m -s', pty=True, warn=True, hide=True)
```

Comment:
I believe that if the venv is created with/for python 3, there is no need to call `python3`, `python` will be aliased to the `python3` executable

Reply:
True, but I like the "expressiveness" of `python3`. YMMV.

Reply:
I’ve been thinking about this, I’ll revert to `python` everywhere, including udata core. 👌